packages/components/src/spinner/style.scss

```diff
@@ -0,0 +1,32 @@
+.components-spinner {
```

Comment:
Sorry to pop in 🙂 Why does this component use a mix of Scss and Emotion? Is it possible for it to just use Emotion? (I think that's the way things are headed)

cc @ciampo

Reply:
> Sorry to pop in 🙂

Welcome! :D I'm using emotion where I need to inject dynamic values (like the spinner size) in to the css, then scss everywhere else. I figured plain scss is more contributor friendly. 

It certainly could be all-emotion if that's preferable.

> Also, if this component replaces the existing spinner, we could probably go ahead and remove the spinner in components/src/ui/spinner as well to avoid any confusion for future contributors to the components package.

Do you think it would be ok to do that in a follow-up? I'm not entirely sure how/where it is used.

Reply:
> It certainly could be all-emotion if that's preferable.

I think @ciampo should be able to answer that. I'm not familiar with the intentions of the components package since I stopped working on it a few months ago. If he doesn't get back to you in a timely manner I'd say a fast-follow PR to convert it to all Emotion would probably be fine.

> Do you think it would be ok to do that in a follow-up? I'm not entirely sure how/where it is used.

Sure thing 👍 FWIW I think it is unused so will hopefully be a straightforward PR (but don't quote me on that! 😅)

Reply:
I think Woo may have used it, so to be safe I'll do that separately :)

Reply:
It doesn't look like the `ui/spinner` is exported from the components package, so hopefully Woo isn't doing something to force it to be available 😱

Reply:
Oh, I just realised that ui/spinner was the original intended replacement for the Spinner that I worked on in this PR (IE that one that is actually published and in use). In which case it would seem very safe to remove ui/spinner. I'll still probably do that separately, though :D

Reply:
Thank you @sarayourfriend for the ping — I haven't taken a deeper look yet, but we would definitely like to [use Emotion to style our components](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/CONTRIBUTING.md#styling).

@jameskoster , would you be able to move back to CSS-in-JS styles (Emotion) in this PR?

The removal of `ui/spinner` can definitely happen in a separate PR

Reply:
> would you be able to move back to CSS-in-JS styles (Emotion) in this PR?

Yup np, I just pushed an update.

Reply:
> Yup np, I just pushed an update.

Thank you! One last thing on this end — would you mind moving the code for the styles (i.e. `spinAnimation`, `StyledSpinner` and `SpinnerIndicator`) to a separate file? `packages/components/src/spinner/styles.js` would be perfect